diff --git a/setup.py b/setup.py
index 57490e3..daa9ab1 100644
--- a/setup.py
+++ b/setup.py
@@ -1,5 +1,5 @@
 import re
-from setuptools import setup, find_namespace_packages
+from setuptools import setup, find_namespace_packages, find_packages
 
 
 with open("README.md", "r") as f:
@@ -21,7 +21,7 @@ setup(
     description="TycheTools raft implementation",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    packages=setuptools.find_packages(),
+    packages=find_packages(),
     install_requires=[
         "msgpack==1.0.2"
     ],
